securedrop/journalist.py

```diff
@@ -195,8 +195,13 @@ def admin_add_user():
                     flash("That username is already in use",
                           "error")
                 else:
-                    flash("An error occurred saving this user to the database",
+                    flash("An error occurred "
```

Comment:
Starting this string on the next line might help condense the number of lines it takes up. Same goes for with `assertMessageFlashed` below.